src/main/java/org/isf/lab/model/Laboratory.java

```diff
@@ -106,10 +106,21 @@ public class Laboratory extends Auditable<String> {
 
 	@Column(name="LAB_SEX")
 	private String sex;
+	
+	@Column(name="LAB_STATUS")
+	private String status;
 
 	@Transient
 	private volatile int hashCode = 0;
 
+	public String getStatus() {
+		return status;
+	}
+
+	public void setStatus(String status) {
+		this.status = status;
+	}
+
```

Comment:
please move getters and setters along with others in the class

Reply:
okay